Formula/fnm.rb

```diff
@@ -1,8 +1,8 @@
 class Fnm < Formula
   desc "Fast and simple Node.js version manager"
   homepage "https://fnm.vercel.app"
-  url "https://github.com/Schniz/fnm/archive/v1.22.9.tar.gz"
-  sha256 "02626f258c4ed57ea60891c2374eb970635ce755b8c5ef22f2076e22840d90cf"
+  url "https://github.com/Schniz/fnm/archive/v1.23.0.tar.gz"
+  sha256 "cf559ce8022605e0b0ddba37db836e16f45baeb1111462b36665aeb6bc438013"
```

Comment:
```suggestion
  sha256 "778221f7309c03d74ed7aac2c6e552e325ab2a85e536029afa8b27d2fa5a22f1"
```

You probably did something like
```
curl https://github.com/Schniz/fnm/archive/v1.23.0.tar.gz | sha256sum
```
when what you want is
```
curl -L https://github.com/Schniz/fnm/archive/v1.23.0.tar.gz | sha256sum
```
The former gives you the checksum of GitHub's redirection stub, not the source tarball.

Reply:
Or, you can automate all of this with
```
brew bump-formula-pr --version=1.23.0 fnm
```